src/transformStateWithClones.js

```diff
@@ -6,8 +6,51 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
-  // write code here
+  // const newObject = {};
+
+  const clone = {}; // новий порожній об'єкт
+  const resultArray = [];
+
+  for (const key in state) {
+    clone[key] = state[key];
+  }
```

Comment:
Instead of copying properties one by one in a loop, you can use the spread operator to make a copy of the `state` object. This is more concise and adheres to the code style guideline of not mutating objects directly.